cmds/ocm/commands/ocicmds/common/handlers/artifacthdlr/attached.go

```diff
@@ -22,9 +22,15 @@ func Attachment(d digest.Digest, suffix string) string {
 var ExplodeAttached = processing.Explode(explodeAttached)
 
 func explodeAttached(o interface{}) []interface{} {
-	obj := o.(*Object)
+	obj, ok := o.(*Object)
+	if !ok {
+		return nil
+	}
 	result := []interface{}{o}
-	blob, _ := obj.Artifact.Blob()
+	blob, err := obj.Artifact.Blob()
+	if err != nil {
+		return result
```

Comment:
although it's better to handle the error (instead of completely ignoring it), shouldn't we log the error at least? Otherwise it might get tricky to identify the root cause, when just an empty result is returned.

Reply:
I can't get the logger into this part, as it would require the ocm context to be passed in.